site/docs/samples/pre-steps.md

```diff
@@ -1,7 +1,7 @@
 # Tools used by the actors
 
-- The data owner would typically register the dataset in a proprietary or open source catalog.  We have provided [katalog](../reference/katalog.md) - a thin layer acting as a replacement for the data catalog for evaluation purposes. This simplifies the sample deployment.
-- The data owner stores credentials for accessing the dataset in kubernetes secrets.
+- The data owner would typically register the dataset in a proprietary or open source catalog. We use [OpenMetadata](https://open-metadata.org/).
+- The data owner needs to provide credentials for accessing a dataset. One way to do it is by storing credentials in kubernetes secrets.

```

Comment:
Doesn't Open Metadata store the credentials for our examples?

Reply:
@shlomitk1 : no, once the secrets are in OpenMetadata, we always return a path the omd plugin.

Reply:
I'm looking at the connector code, and it's so confusing. The credentials (path to k8s secret) are translated to the actual secret key and access key but are stored as-is as a common property (Credentials). In GetAssetInfo first the common property is attempted to be returned but then it is overridden by the path to omd plugin. If the "real" secret fields are stored in OM, what is the meaning of Credentials property?

Reply:
removing the `credentials` custom property is on my to-do list :)
https://github.com/fybrik/openmetadata-connector/issues/32

Reply:
cool, thanks